pkgs/development/mobile/androidenv/test-suite.nix

```diff
@@ -1,16 +1,20 @@
-{ stdenv, callPackage }:
+{}:
 let
-  examples-shell = callPackage ./examples/shell.nix {};
+  pkgs = import ../../../.. {};
```

Comment:
Why do we import nixpkgs here?

Reply:
To make sure the tests are only depending on the current reversion of the repository, and avoid injecting other possible ones! Does this make sense? :D If not, no problem! I'm going to remove this.

Reply:
We can't use callPackage to inherit the already evaluated packages?

Reply:
done in [e6ecb9f](https://github.com/NixOS/nixpkgs/pull/213871/commits/e6ecb9f2c97dc68d379c600c91bf0d25bbc2d881). Let me know if it's okay so I squash my commits.